index.css

```diff
@@ -91,7 +91,9 @@ body {
 }
 
 .intro h2 {
-  color: #ec5242;
+  background: -webkit-linear-gradient(60deg, #f71d13, #b70606, #f75515, #f75515, #f75515);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
   width: 260px;
   line-height: 40px;
```

Comment:
- [ ] Kindly ensure that the `footer` section of your `about me ` section  has a darker background like in the original design

![image](https://user-images.githubusercontent.com/98453979/219084106-1372d698-45b7-47a9-ae66-2faa3ee8da4d.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/219084344-3d84716d-9337-4b18-b964-004bdf58c729.png)

Reply:
The previous reviewer wanted you to use the dark background shown in the image above :arrow_up: but the color you are using is `#808080` you are supposed to use `#2e2e2e`.